coordinatorlib/src/main/scala/com/socrata/datacoordinator/secondary/PlaybackToSecondary.scala

```diff
@@ -114,7 +116,7 @@ class PlaybackToSecondary[CT, CV](u: PlaybackToSecondary.SuperUniverse[CT, CV],
 
   def apply(secondary: NamedSecondary[CT, CV], job: SecondaryRecord): Unit = {
     if (job.pendingDrop) {
-      logger.info("Executing pending drop of dataset {} from store {}.", job.datasetId.toString, job.storeId)
+      logger.info("Executing pending drop of dataset {} from store {}.", job.datasetId : Any, job.storeId)
```

Comment:
This is a side effect of de-scalalogging the class (this was IIRC the only place in the codebase where scalalogging was used, so I removed it).  Unfortunately, scalac has a hard time resolving the overload for slf4j logging lines in the case of precisely values to be formatted into the log line, so I need to give it a hint.

The `toString` went away because it was unnecessary.  `toString` will get called on it when it gets interpolated.